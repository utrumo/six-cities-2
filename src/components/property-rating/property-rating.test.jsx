import React from 'react';
import renderer from 'react-test-renderer';
import PropertyRating from './property-rating.jsx';

it(`PropertyRating should renders correctly`, () => {
  const tree = renderer.create(<PropertyRating rating={3.2} />).toJSON();
  expect(tree).toMatchSnapshot();
});
