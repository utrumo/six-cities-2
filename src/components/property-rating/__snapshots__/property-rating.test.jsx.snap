// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PropertyRating should renders correctly 1`] = `
<div
  className="property__rating rating"
>
  <div
    className="property__stars rating__stars"
  >
    <span
      className="rating__gotten-stars"
      style={
        Object {
          "width": "60%",
        }
      }
    />
    <span
      className="visually-hidden"
    >
      Rating
    </span>
  </div>
  <span
    className="property__rating-value rating__value"
  >
    3
  </span>
</div>
`;
